homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -121,7 +123,7 @@ def binary_sensor_update(event):
             sensor = RfxtrxBinarySensor(event, pkt_id)
             sensor.apply_event(event)
             RFX_DEVICES[device_id] = sensor
-            add_entities([sensor])
+            hass.async_add_job(add_entities, [sensor])
```

Comment:
We should use `async_add_entities` and `async_setup_platform` if we want to start using the async api.

I think this whole change can be a later PR.